pkg/qiniu/object.go

```diff
@@ -29,8 +29,14 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// ObjectHandle provides operations on an object in a qiniu cloud bucket
-type ObjectHandle struct {
+type ObjectHandle interface {
```

Comment:
suggest to add comments for exported interface

Reply:
added~